test/python/test_secrets_api.py

```diff
@@ -266,17 +266,21 @@ def test_get_variables_422(self):
 
         self.assertEqual(context.exception.status, 422)
 
-    def test_get_variables_encoded(self):
+    def test_get_variables_encoded(self): # pylint: disable=R1710
         """Test case for get_variables 200 response with base64 encoded binary secrets"""
         secret_values = [b'one\xffbinary', b'two']
         self.set_variables(TEST_VARIABLES, secret_values)
 
         # Secrets have to be in the format org:variable:secret_name
-        secret_list = [f"dev:variable:{i}" for i in TEST_VARIABLES]
-        response, status, headers = self.api.get_variables_with_http_info(
-            ",".join(secret_list),
-            accept_encoding="base64"
-        )
+        secret_list = [f"{self.account}:variable:{i}" for i in TEST_VARIABLES]
+        try:
+            response, status, headers = self.api.get_variables_with_http_info(
+                ",".join(secret_list),
+                accept_encoding="base64"
+            )
+        except openapi_client.ApiException as error:
+            if error.status == 500 and api_config.DAP_TESTS:
+                return unittest.skip("DAP not up to date with latest Conjur version")
```

Comment:
```suggestion
            if error.status == 500 and api_config.ENTERPRISE_TESTS:
                return unittest.skip("Enterprise not up to date with latest Conjur OSS version")
```

Reply:
is this really a test we have to persist? this is an interesting case, we may have to think about how to handle version mismatches more elegantly going forward.

Reply:
We do have an edge case test for this in Conjur itself, so we don't necessarily need to test for functionality here, we could just use a unit test to check that the header is included in the request. It will be difficult to deal with version mismatches, no real good way to deal with it comes to mind right now but I will put some thought into it